sklearn/metrics/_scorer.py

```diff
@@ -432,6 +432,7 @@ def check_scoring(estimator, scoring=None, *, allow_none=False):
         A string (see model evaluation documentation) or
         a scorer callable object / function with signature
         ``scorer(estimator, X, y)``.
+        If None, the provided estimator object's 'score' method is used.
```

Comment:
Sorry for being unclear, I think it should be backticks, which will make it be rendered as code. ```

Reply:
no problem, i fixed it!